Makefile

```diff
@@ -188,6 +200,43 @@ SRC_C += nrfx/drivers/src/prs/nrfx_prs.c
 SRC_C += nrfx/helpers/nrfx_flag32_allocator.c
 SRC_C += nrfx/mdk/system_nrf52.c
 
+SRC_C += speex/libspeex/bits.c
+SRC_C += speex/libspeex/cb_search.c
+SRC_C += speex/libspeex/exc_10_16_table.c
+SRC_C += speex/libspeex/exc_10_32_table.c
+SRC_C += speex/libspeex/exc_20_32_table.c
+SRC_C += speex/libspeex/exc_5_256_table.c
+SRC_C += speex/libspeex/exc_5_64_table.c
+SRC_C += speex/libspeex/exc_8_128_table.c
+SRC_C += speex/libspeex/fftwrap.c
+SRC_C += speex/libspeex/filters.c
+SRC_C += speex/libspeex/gain_table.c
+SRC_C += speex/libspeex/gain_table_lbr.c
+SRC_C += speex/libspeex/hexc_10_32_table.c
+SRC_C += speex/libspeex/hexc_table.c
+SRC_C += speex/libspeex/high_lsp_tables.c
+SRC_C += speex/libspeex/kiss_fft.c
+SRC_C += speex/libspeex/kiss_fftr.c
+SRC_C += speex/libspeex/lpc.c
+SRC_C += speex/libspeex/lsp.c
+SRC_C += speex/libspeex/lsp_tables_nb.c
+SRC_C += speex/libspeex/ltp.c
+SRC_C += speex/libspeex/modes.c
+SRC_C += speex/libspeex/modes_wb.c
+SRC_C += speex/libspeex/nb_celp.c
+SRC_C += speex/libspeex/quant_lsp.c
+SRC_C += speex/libspeex/sb_celp.c
+SRC_C += speex/libspeex/smallft.c
+SRC_C += speex/libspeex/speex.c
+SRC_C += speex/libspeex/speex_callbacks.c
+SRC_C += speex/libspeex/speex_header.c
+SRC_C += speex/libspeex/stereo.c
+SRC_C += speex/libspeex/vbr.c
+SRC_C += speex/libspeex/vorbis_psy.c
+SRC_C += speex/libspeex/vq.c
+SRC_C += speex/libspeex/window.c
+SRC_C += os_support_custom.c
```

Comment:
This could be renamed however we want, here made to be matching `os_support_custom.h`, name hardcoded in libspeex.